i18n/es.json

```diff
@@ -208,7 +208,7 @@
   "I18N_PLAYER_RATE_EXPLORATION": "¿Has aprendido algo nuevo? ¿Cómo calificarías esta exploración?",
   "I18N_PLAYER_CARD_NUMBER_TOOLTIP": "Carta #",
   "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "Otras exploraciones que tal vez te gusten",
-  "I18N_PLAYER_RETURN_TO_GALLERY": "Volver a la galería",
+  "I18N_PLAYER_RETURN_TO_LIBRARY": "Volver a la librería",
```

Comment:
I got a different translation according to Google Translate (with all its goings), can you confirm this translation?